kolibri/plugins/user_profile/tasks.py

```diff
@@ -111,7 +111,7 @@ def mergeuser(command, **kwargs):
     merge_users(local_user, remote_user)
 
     # Resync with the server to update the merged records
-    # kwargs["no_pull"] = True
+    kwargs["no_pull"] = True
```

Comment:
While it is unlikely that an issue would arise from this, having this set to `no_pull` does leave a possibility.